src/test/java/org/alliancegenome/mati/controller/SubdomainResourceITCase.java

```diff
@@ -29,13 +29,4 @@ public void getAll() {
             .then()
             .statusCode(200);
     }
-
-    @Test
-    public void getOne() {
-        given()
-            .when()
-            .get("/api/subdomain/1")
-            .then()
-            .statusCode(200);
-    }
```

Comment:
@abecerra Why would this integration test not be relevant? I would assume this integration test needs fixing rather than removal?

Reply:
I am deactivating the information about individual subdomains, and only leaving the getAll

Reply:
Why though, I'm sure being able to retrieve info about an individual subdomain without needing to fetch all and filter the results is useful to users? Also, you didn't remove the endpoint did you, only the integration test? From my point of view, this endpoint still needs testing as long as we provide it to users?

Reply:
The endpoints for subdomain are automatically generated by panache-rest. The one to get the subdomain per id is hidden now. The idea is that users (developers in teams) will be the ones fetching all the subdomain ids to use the proper id when calling MaTI

The hiding: 
https://github.com/alliance-genome/agr_mati/blob/428d8d75242b9ed0d21ab6c4699e602bbc2d5ffe/src/main/java/org/alliancegenome/mati/controller/SubdomainResource.java#L25